src/hnsw/utils.c

```diff
@@ -177,6 +177,17 @@ bool VersionsMatch()
         version_checked = true;
 
         SPI_finish();
+
+        if(!versions_match) {
+            elog(WARNING,
```

Comment:
Do we need the elog to be moved here? As now we will have duplicate logs for build and insert (as they're changed to WARNING) right?

Reply:
this is logged once, as otherwise `versions_match` is cached.
it is good to have here for debugging, since it actually prints the version names and tells something about what the mismatch is